projects/angular/src/wizard/wizard.html

```diff
@@ -14,15 +14,22 @@
   [clrModalLabelledById]="wizardId"
 >
   <nav class="modal-nav clr-wizard-stepnav-wrapper">
-    <h1 class="clr-wizard-title" id="{{wizardId}}"><ng-content select="clr-wizard-title"></ng-content></h1>
+    <ng-content select="clr-wizard-title"></ng-content>
```

Comment:
Can we move the `clr-wizard-title` CSS class to a wrapper span/div so that users don't have to include that class on custom heading elements?

Reply:
Same thing for the `id`.